### PR TITLE
Move to dcos-commons CustomExecutor

### PR DESCRIPTION
**Pull request by gabrielhartmann**

### CODE REVIEW

**Comment by gabrielhartmann** [comment]
@mohitsoni:  Please review.  I know it doesn't build in CI, I'll take care of that after Wednesday September 28.  I have run integration and unit tests manually...

**Comment by gabrielhartmann** [comment]
@mesosphere-ci: retest this please

**Comment by gabrielhartmann** [comment]
@mesosphere-ci: retest this please

File: cassandra-executor/src/main/java/com/mesosphere/dcos/cassandra/executor/CassandraExecutor.java [hunk 1]
```diff
@@ -47,128 +38,10 @@
 public class CassandraExecutor implements Executor {
```
Comment (gabrielhartmann) [inline cassandra-executor/src/main/java/com/mesosphere/dcos/cassandra/executor/CassandraExecutor.java]: It still has one method that's needed the `getCassandraDaemon()`.  I'm trying to change as little as possible in this PR.  Ideally Cassandra will adopt the SDK soon and at that point the need for the CassandraExecutor will go away entirely.

Comment (mohitsoni) [inline cassandra-executor/src/main/java/com/mesosphere/dcos/cassandra/executor/CassandraExecutor.java]: Okay, makes sense.

File: cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/offer/ClusterTaskPlacementStrategy.java [hunk 2]
```diff
@@ -2,28 +2,25 @@
...
     }
 
-    @Override
-    public List<Protos.SlaveID> getAgentsToColocate(Protos.TaskInfo taskInfo) {
-        List<Protos.SlaveID> agentsToColocate = new ArrayList<>();
+    public static List<String> getAgentsToColocate(Protos.TaskInfo taskInfo) {
+        List<String> agentsToColocate = new ArrayList<>();
         // Collocate this task with the corresponding Cassandra node task
-        Arrays.asList(taskInfo.getSlaveId());
+        Arrays.asList(taskInfo.getSlaveId().getValue());
```
Comment (verma7) [inline cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/offer/ClusterTaskPlacementStrategy.java]: As someone pointed out https://github.com/mesosphere/dcos-cassandra-service/issues/229, should this line be 
`agentsToColocate(taskInfo.getSlaveId().getValue())`?

Comment (gabrielhartmann) [inline cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/offer/ClusterTaskPlacementStrategy.java]: Actually ClusterTaskPlacementStrategy hasn't had any impact in a long time.  Deleted the file.  Thanks for pointing this out.

**Comment by gabrielhartmann** [comment]
@mesosphere-ci: retest this please

**Comment by gabrielhartmann** [comment]
@mesosphere-ci: retest this please

**Comment by gabrielhartmann** [comment]
@mesosphere-ci: retest this please

**Comment by gabrielhartmann** [comment]
@mesosphere-ci: retest this please

**Comment by gabrielhartmann** [comment]
@mesosphere-ci: retest this please

**Comment by gabrielhartmann** [comment]
@mesosphere-ci: retest this please

**Pull request merged**